app/view_models/v1/letter_and_call.rb

```diff
@@ -62,6 +80,23 @@ def table_fields
       ]
     end
 
+    def provider_fields
+      rows = {
+        '.number' => provider_requested_count.to_s,
+        '.rate' => NumberTo.pounds(pricing),
+        '.uplift_requested' => "#{provider_requested_uplift.to_i}%",
+      }
+
+      if vat_registered?
+        rows['.vat'] = NumberTo.percentage(vat_rate)
+        rows['.total_claimed_inc_vate'] = NumberTo.pounds_inc_vat(provider_requested_amount_inc_vat)
```

Comment:
**nit** feels a bit odd to be specifying `..inc_vat` twice.

`NumberTo.pounds(provider_requested_amount_inc_vat, round: :down)` feels more declarative of intent,  but I can see the advantage when called in multiple other places and can think of anything better so 👍

Reply:
I have to rethink all of this anyway as realised it rounds incorrectly (I think) when firm is not vat registered - somethings... :( 

I will have a rethink of the approach when I start looking at that - have a meeting to confirm how rounding should work tomorrow morning. :)